source/api/extension/navplace/index.md

```diff
@@ -21,14 +21,14 @@ Changes will be tracked within the document.
 
 ## 1. Introduction
 
-The ideas of Time and Place are a foundation of human communication and understanding. These concepts are first class descriptors for earthbound resources and the ability to capture both independently is often required for data presentation and preservation. In IIIF Presentation API 3 the property `navDate` allows for a calendar date to be associated with a resource in a client friendly format. However, places are extents in space, independent of time and what may or may not be present in that space. 
+The ideas of Time and Place are a foundation of human communication and understanding. These concepts are first class descriptors for resources and the ability to capture both independently is often required for data presentation and preservation. In IIIF Presentation API 3 the property `navDate` allows for a calendar date to be associated with a resource in a client friendly format. However, locations are extents in space, independent of time and what may or may not be present in that space. 
 
 
 ### 1.1 Objectives and Scope
 
 IIIF provides the information necessary to allow a rich, online viewing environment for compound digital objects to be presented to a human user. The types of resources and their specific needs continue to enhance the IIIF specifications to greater resource coverage. The ability to assert ancillary information is a universal need in this expanse. This is achieved using Annotation, or Web Annotation, and extensions across the IIIF specification versions.
 
-This extension describes a property called `navPlace` which contains geographic coordinates in the form of [GeoJSON-LD](https://geojson.org/geojson-ld/). Clients may use this property to leverage the navigational functionality of web maps such as Google Earth, Leaflet, OpenLayers, etc. giving them the opportunity to enrich data presentation through common web map platforms.
+This extension describes a property called `navPlace` which contains earthbound geographic coordinates in the form of [GeoJSON-LD](https://geojson.org/geojson-ld/). Clients may use this property to leverage the navigational functionality of web maps such as Google Earth, Leaflet, OpenLayers, etc. giving them the opportunity to enrich data presentation through common web map platforms.
```

Comment:
How in the heck do we justify 'earthbound' in one sentence.

Reply:
Technically b/c GeoJSON and WGS84
Philosophically because...TODO

Reply:
Mention how ambiguous it is to use coordinates in space (we don't know those other celestial spheres as well as Earth).  We know how to measure Earth b/c we are on it and worked on it for centuries.  We just don't have this for space (star maps).  Frame it as a matter of 'universally' agreed upon standards.

Reply:
https://lunar.gsfc.nasa.gov/library/LunCoordWhitePaper-10-08.pdf...but there is no GeoJSON built for this (A GeoJSON that accepts other coordinate systems.)